index.js

```diff
@@ -600,6 +598,30 @@ function hasEncryptionScheme(mediaKeySystemAccess) {
   return false;
 }
 
+/**
+ * @param {?string} scheme Encryption scheme to check
+ * @param {?string} supportedScheme A guess at the encryption scheme this
+ *   supports.
+ * @return {boolean} True if the scheme is compatible.
+ */
+function checkSupportedScheme(scheme, supportedScheme) {
+  if (scheme && scheme != supportedScheme) {
+    if (scheme == 'cbcs') {
+      // Firefox supports CBCS since version 100, but does not yet implement
+      // encryptionScheme support.
+      if (parseInt(navigator.userAgent.split('Firefox/').pop(), 10) >= 100) {
```

Comment:
I would like to see this pulled out to a place where it is run once and cached/memoized.

Reply:
Done!